blas/unit_test/Test_Blas3_gemm.hpp

```diff
@@ -372,12 +371,17 @@ void test_gemm() {
       }
     }
   }
-  Test::impl_test_stream_gemm<Scalar, Layout>(53, 42, 17, 4.5,
-                                              3.0);  // General code path
-  Test::impl_test_stream_gemm<Scalar, Layout>(
-      13, 1, 17, 4.5, 3.0);  // gemv based gemm code path
-  Test::impl_test_stream_gemm<Scalar, Layout>(7, 13, 17, 4.5,
-                                              3.0);  // dot based gemm code path
+  auto pool_size = TestExecSpace().impl_thread_pool_size();
```

Comment:
@lucbv, @vqd8a: Is there a non-impl way to determine how many times an execution space can be partitioned at runtime?

Reply:
Yes, ``TestExecSpace().concurrency()`` is equivalent (for the exec spaces that actually have impl_thread_pool_size, so Serial, Threads and OpenMP). I had to make this change somewhere in #1666 when impl_thread_pool_size became non-static only